.github/workflows/k8s_e2e.yaml

```diff
@@ -0,0 +1,69 @@
+name: Kubernetes e2e testing
+
+env:
+  EXTERNAL_REGISTRY: true
+  REGISTRY_PORT: 80
+
+on:
+  # push:
+  #   paths-ignore:
+  #     # - '.github/**'
+  #     - '.gitignore'
+  #     - 'LICENSE'
+  #     - 'README*'
+  #   branches: ['*']
+    # branches: [master]
+#TODO choose to fire up with cron or on push to master
+# on:
+#   schedule:
+#     - cron:  '0 */2 * * *'
+
+#This trigger is only for testing
+  pull_request:
+    paths-ignore:
+      - '.github/project.yaml'
```

Comment:
`.github/**` would be a better pattern. Also, add `docs/**` to avoid triggering the workflow for any changes in the docs